modules/Emails/include/ComposeView/EmailsComposeView.js

```diff
@@ -1396,6 +1397,8 @@
       convert_urls: true,
       relative_urls: false,
       remove_script_host: false,
+      relative_urls: false,
```

Comment:
Hi @SinergiaCRM I believe you have some duplicate code here, the lines you have added are already there above the lines you added.

Reply:
Hi @jack7anderson7, you are right. 

We have also seen that the proposed solution is not good since now the schema and the domain are added to any URL that is indicated, even in the tracking URLs.

We are going to close this PR for being wrong and open another one with the following solution and configuration:

`convert_urls=false`

Thanks!

Reply:
Hi @jack7anderson7, you are right.

We have also seen that the proposed solution is not good since now the schema and the domain are added to any URL that is indicated, even in the tracking URLs.

We are going to close this PR for being wrong and open another one with the following solution and configuration:

`convert_urls=false`

Thanks!

Reply:
https://github.com/salesagility/SuiteCRM/pull/9777